### PR TITLE
Added link to PyPi to release notes

### PR DESCRIPTION
**Pull request by levius0**

### CODE REVIEW

File: pyepp/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """
 PyEPP Package
 """
-__version__ = "0.0.15"
+__version__ = "0.0.16"
```
[the remark this answers is not in the document]
Comment (levius0) [reply pyepp/__init__.py:4]: If we don't it will cause an error in create tag step.

[the remark this answers is not in the document]
Comment (ehsan-fj) [reply pyepp/__init__.py:4]: if we don't need to release a new version we won't need to create a tag! let's have a chat about it

[the remark this answers is not in the document]
Comment (levius0) [reply pyepp/__init__.py]: I've changed the conditions for deployment, tag creation and release generation; please take a look when you have the time.

**Comment by levius0** [comment]
now everything is ready for the review

File: .github/workflows/deployment.yml [hunk 2]
```diff
@@ -43,27 +51,19 @@ jobs:
         twine check dist/*
         twine upload -r testpypi --username __token__ --password ${{ secrets.TEST_PYPI_API_TOKEN }} --skip-existing dist/*
     - name: Publish to PyPi
-      if: github.event_name == 'push' && github.ref == 'refs/heads/main'
+      if: github.event_name == 'push' && github.ref == 'refs/heads/main' && steps.previous_tag.outputs.tag < env.NEW_VERSION
```
Comment (ehsan-fj) [inline .github/workflows/deployment.yml:54]: Does `<` operation work properly with the version numbers? was it enough to just use `!=`?

File: .github/workflows/deployment.yml [hunk 3]
```diff
@@ -89,13 +89,16 @@ jobs:
...
       uses: ncipollo/release-action@v1
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
-        tag: ${{env.NEW_VERSION}}
+        tag: ${{ env.NEW_VERSION }}
         draft: false
-        body: ${{steps.build_changelog.outputs.changelog}}
+        body: |
+          [PyEPP on PyPI](https://pypi.org/project/pyepp/)
```
Comment (ehsan-fj) [inline .github/workflows/deployment.yml:99]: Can you contact the version number at the end of this URL, like `https://pypi.org/project/pyepp/0.0.15/` so if someone goes back to releases, they can find the right URL for the version?

File: .github/workflows/deployment.yml (hunk 2 above)
Comment (levius0) [reply .github/workflows/deployment.yml:54]: It is, I checked it yesterday. But I will test it once again today.

File: .github/workflows/deployment.yml (hunk 3 above)
Comment (levius0) [reply .github/workflows/deployment.yml:99]: ok, I will add it. thanks

File: .github/workflows/deployment.yml (hunk 2 above)
Comment (levius0) [reply .github/workflows/deployment.yml:54]: I decided to use < because it may prevent an accidental decrease of the verions

Comment (levius0) [reply .github/workflows/deployment.yml:56]: Checked once again it works correctly

**Review by ehsan-fj — APPROVED** [review]

**Pull request merged**